src/Microsoft.EntityFrameworkCore.Relational/Properties/RelationalStrings.resx

```
@@ -193,7 +193,7 @@
     <value>Unable to set property '{property}' as a discriminator for entity type '{entityType}' because it is not a property of '{entityType}'.</value>
   </data>
   <data name="AmbientTransaction" xml:space="preserve">
-    <value>An ambient transaction has been detected. Connections opened by Entity Framework will not be enlisted in ambient transactions. To suppress this warning call SuppressAmbientTransactionWarning() when overriding DbContext.OnConfiguring.</value>
+    <value>An ambient transaction has been detected. Entity Framework Core does not support ambient transactions. To suppress this warning call ConfigureWarnings when overriding DbContext.OnConfiguring.</value>
```

:+1: for first sentence. It fixes #4147. 

For second sentence
- I believe it should say `To suppress this exception use the ConfigureWarnings API` like in the previous one, assuming we switched it back to throw by default. 
- Same concern as before that many customers will call this in `AddDbContext()` rather than overriding `OnConfiguring()`.

Still not sure about second sentence.